clients/src/main/resources/common/message/EndQuorumEpochRequest.json

```diff
@@ -26,6 +26,8 @@
     {"name": "LeaderId", "type": "int32", "versions": "0+",
       "about": "The current leader ID or -1 if there is a vote in progress"},
     {"name": "LeaderEpoch", "type": "int32", "versions": "0+",
-      "about": "The current epoch"}
+      "about": "The current epoch"},
+    {"name": "PreferableVoters", "type": "[]int32", "versions": "0+",
```

Comment:
How about `preferredSuccessors` or `preferredCandidates`?

Reply:
Successors sound better to me :)